sap-s4hana-cloud/2.2/modules/ROOT/pages/sap-s4hana-cloud-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,80 @@
+= Upgrade SAP S/4HANA OData Connector to 2.x - Mule 4
+:page-aliases: connectors::sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc
+
+Upgrade Anypoint Connector for SAP S/4HANA OData (SAP S/4HANA OData Connector) to take advantage of the increased security offered by using OAuth 2.0 authentication to connect to SAP S/4HANA OData.
+
+== Supported Upgrade Paths
+
+This is a direct upgrade from version 1.2.1 version to 2.x.x.
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.2.1 |2.x.x
+|===
+
+== Changes in Version 2.0.0
+
+* OAuth 2.0 is supported. +
+The *OAuth 2.0 Application Connection Provider* option has been added to *Connection* types. For information about the configuration parameters for this connection type, see xref:sap-s4hana-cloud-connector-reference.adoc[SAP S/4HANA OData Connector Reference].
+* Mule runtime engine (Mule) 4.2.1 or later is required.
+* The default value for the *Service directory* field has been removed.
+* Because the base package has been changed, all exported classes now have different fully qualified names.
+* The group ID of the connector’s JAR has changed.
+
+== Changes in 2.2.1
```

Comment:
I like that you separated the changes for each release here. It's a good idea. However, I would put these sections as subsections of a section called "What's Changed" or something like that. The reason is that if there are a lot of minor releases, this could get unwieldy to read. I would consider putting the latest version first because it is probably the one users would be more likely to look for.

It's fine to leave what you have if you want to - we can revisit this after I get back.